vendor/assets/javascripts/brainstem/brainstem-model.coffee

```diff
@@ -95,6 +105,16 @@ class window.Brainstem.Model extends Backbone.Model
     else
       super(field)
 
+  set: (key, val, options) ->
+    super
+
+    _.extend(@unsavedAttributes || {}, @changed)
```

Comment:
Sort of :)

Would switching `super` and `@unsavedAttributes = {}` in the constructor let us remove the `||` here?